CMSgov/onemac
# OY2-2524 update about page

**Story:** https://qmacbis.atlassian.net/browse/OY2-2524

**Application endpoint:** https://d3tmapsde7kivw.cloudfront.net/

**Mock up:**
![image](https://user-images.githubusercontent.com/40372898/98995821-ec9e7480-24ff-11eb-8f23-b917cb4fda21.png)

### Changes
- Updated content on about page
- Cleaned up/repurposed functions for displaying bulleted list
- Cleaned up styling 

### Testing
- Check that subtitle in header content reflects attached mock up
- Check that the "In this system" section displays updated content per mock up in bulleted list format
